src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -449,7 +460,7 @@ protected int doWork() {
             if (SequenceUtil.isNoCall(base)) continue;
 
             // add to the collector
-            collector.addInfo(info, ref);
+            collector.addInfo(info);
```

Comment:
this is a publicly available method, I would hesistate a little longer before changing the signature. Do you _need_ to change it or is this a perceived optimization?